curses-client.py

```diff
@@ -79,6 +79,8 @@
 selected_channel = 0
 selected_node = 0
 direct_message = False
+packet_buffer = []
+display_log = False
 
 def get_channels():
     global channel_list
```

Comment:
needed for suggestion around line 294
```suggestion
    global channel_list
    global modem_preset_string
```